project/Dependencies.scala

```diff
@@ -17,11 +19,15 @@ object Dependencies {
       "com.typesafe.akka" %% "akka-cluster-sharding"              % AkkaVersion,
       "com.typesafe.akka" %% "akka-http"                          % AkkaHttpVersion,
       "com.typesafe.akka" %% "akka-http-spray-json"               % AkkaHttpVersion,
-      "io.spray"          %% "spray-json"                         % "1.3.3",                  // ApacheV2
+      "io.spray"          %% "spray-json"                         % SprayJsonVersion,                  // ApacheV2
+
+      "ru.smslv.akka"     %% "akka-dns"                           % akkaDns,
```

Comment:
I get unresolved dependency: ru.smslv.akka#akka-dns_2.11;2.4.2: not found

Reply:
Whoa weird, where did I get the artifact from hm...